recipes/libsodium/all/conandata.yml

```diff
@@ -2,7 +2,13 @@ sources:
   "1.0.18":
     url: "https://github.com/jedisct1/libsodium/releases/download/1.0.18-RELEASE/libsodium-1.0.18.tar.gz"
     sha256: "6f504490b342a4f8a4c4a02fc9b866cbef8622d5df4e5452b46be121e46636c1"
+  "cci.20220501":
+    url: "https://github.com/jedisct1/libsodium/archive/78b6f57493a02a70fca44e5ef3f9d7d1cae9029f.tar.gz"
```

Comment:
The date should follow the commit date. This commit is not listed on 2022/05/01 https://github.com/jedisct1/libsodium/commits/master

Reply:
You must look at the stable branch.
The master branch is their development branch, and they cherry-pick into the stable - it is a massive fork.
Very weird pattern.
https://github.com/jedisct1/libsodium/commits/stable
You'll find the commit there.
Should I add a comment about that in the conandata.yml file?

Reply:
that commit is from April, not May. Update the date: https://github.com/jedisct1/libsodium/tree/78b6f57493a02a70fca44e5ef3f9d7d1cae9029f

Reply:
Oh wow, i looked at this on github and took the date from there: 
![image](https://user-images.githubusercontent.com/1646667/169324905-26c14ecc-7e30-412a-a4af-06efacc3edec.png)

I'll update the date